### PR TITLE
Create New Delhi recommendation file by bhediathewolf

### PR DESCRIPTION
**Pull request by bhediathewolf**
This branch,
- will create a recommendation for New Delhi, India.

### CODE REVIEW

**Comment by nrprise** [comment]
Here's your general comment

File: bhediathewolf-NewDelhi.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# New Delhi
```
Comment (nrprise) [inline bhediathewolf-NewDelhi.md:1]: Lots of detail. Great job!

Comment (bhediathewolf) [reply bhediathewolf-NewDelhi.md:1]: Thank you!

**Review by nrprise — APPROVED** [review]
Good job. Makes me really look forward to a visit to New Delhi!

**Comment by bhediathewolf** [comment]
Lucky to have you here!

**Pull request merged**